### ISSUE TITLE
[transport-ble] Discussions on implementation

### ISSUE THREAD

**Issue by rapaktech** [hyperledger/aries-framework-javascript-ext#197]
The BLE transport is still in its early stages. Discussions on the future trajectory of the project are held here, to be moved to a better location in the future if necessary.

**Comment by tbloomfi** [comment]
This is not specific to the implementation of the transport which is a great step forward but a more general observation on the implementation of working solutions. 

Without a precise specification, implementations will not be interoperable. See the ISO eID/mDL and the very early draft of the OpenID4VCP over BLE for examples of the level of detail I am thinking of. Aries either needs to adapt to one of the above-listed specs or develop an RFC that provides the level of precision required by vendors.

Device vendors are going to build to the ISO spec as a default stance - NFC/QR engagement with BLE transport.

**Comment by TimoGlastra** [comment]
Good point Tim! For now we've mostly been targeting https://github.com/decentralized-identity/didcomm-bluetooth/blob/main/spec.md. 

Once we have a fully working implementation, we'll contribute back to that specification, revise it and make sure we learn from the OpenID4VP over BLE
